tests/mock_crust/churn.rs

```diff
@@ -93,8 +93,27 @@ impl ExpectedGets {
         self.messages.insert((data_id, msg_id, src, dst));
     }
 
+    /// Sends a request from the client, and adds the expectation that the nodes belonging to `dst`
```

Comment:
I think the comment should make it clear that the function is sending a `Get` request from only the first client in the slice, or the function should be changed to only take a single `TestClient` rather than a slice of them.